Script/AtomicEditor/hostExtensions/languageExtensions/TypscriptLanguageExtension.ts

```diff
@@ -337,6 +341,42 @@ export default class TypescriptLanguageExtension implements Editor.HostExtension
         WebView.WebBrowserHost.setGlobalStringProperty("TypeScriptLanguageExtension", "tsConfig", JSON.stringify(tsConfig));
     }
 
+    generateExternalProject ()
+    {
+      var projectPath : string = ToolCore.toolSystem.project.projectPath;
+
+      // Create TypeScript folder in project root
+      var tsPath : string = projectPath + "TypeScript/";
+      Atomic.getFileSystem().createDir(tsPath);
+
+      // Copy the Atomic.d.ts definition file to the TypeScript folder
+      var toolDataDir : string = ToolCore.toolEnvironment.toolDataDir;
+      Atomic.getFileSystem().copy(toolDataDir + "/TypeScriptSupport/Atomic.d.ts", tsPath + "Atomic.d.ts");
```

Comment:
In practice "/" and "\" are pretty interchangeable on Windows, though if there is a problem we should modify methods like copy to use the native path representation, instead of making calling sites responsible for handling it